metacat-main/build.gradle

```diff
@@ -76,9 +77,10 @@ dependencies {
     runtime(project(":metacat-connector-druid"))
 
     runtime("io.springfox:springfox-swagger-ui")
-    runtime("org.webjars:hal-browser")
+    runtime("org.webjars:hal-browser:3325375")
```

Comment:
Curious what the reason for this pinning is

Reply:
The new Netflix BOm does not have recommendations for this artifact, so we have to specify a specific version